### PR TITLE
Adds global gstore config to return null on entity not found

### PR DESCRIPTION
**Pull request by huysamen**

### CODE REVIEW

File: test/model-test.js [hunk 1]
```diff
@@ -434,6 +445,18 @@ describe('Model', () => {
                     });
             });
 
+            it('should throw an Error if entity not found in cache', (done) => {
```
Comment (sebelga) [inline test/model-test.js]: This should be "should return null if entity not found in cache" no?

File: lib/index.d.ts [hunk 2]
```diff
@@ -612,7 +612,8 @@ declare namespace GstoreNode {
    */
   interface GstoreConfig {
     namespace?: string;
-    cache?: Boolean | CacheConfig;
+	cache?: Boolean | CacheConfig;
```
Comment (sebelga) [inline lib/index.d.ts]: Could you fix indentation :)

**Comment by huysamen** [comment]
Done.

**Comment by sebelga** [comment]
Thank you!

**Pull request merged**